setup.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+from setuptools import setup, find_packages  # Always prefer setuptools over distutils
+from codecs import open  # To use a consistent encoding
+from os import path
+
+here = path.abspath(path.dirname(__file__))
+
+# Get the long description from the relevant file
+with open(path.join(here, 'README.md'), encoding='utf-8') as f:
+    long_description = f.read()
+
+
+setup(
+    name='ckan3-py-sdk',
+
+    # Versions should comply with PEP440.  For a discussion on single-sourcing
+    # the version across setup.py and the project code, see
+    # http://packaging.python.org/en/latest/tutorial.html#version
+    version='0.1.0',
+
+    description='''Ckan3-py-sdk is a "SDK" in Python for uploading files and updating metastore.''',
```

Comment:
Usually this would be auto-pulled from README ...

If you were doing this i would have:

```
CKAN3 Python SDK is a Python SDK for CKAN with a focus on ...
```